lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroup.kt

```diff
@@ -57,10 +58,7 @@ abstract class LibGroup(
         encodeDefaults = true
     }
 
-    private val preferences: SharedPreferences by lazy {
-        Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
-            .migrateOldImageServer()
-    }
+    private val preferences by getPreferencesLazy { migrateOldImageServer() }
```

Comment:
And remove the code below
```suggestion
    private val preferences by getPreferencesLazy { 
        if (getString(SERVER_PREF, "main") == "fourth") {
            edit().putString(SERVER_PREF, "secondary").apply()
        }
    }
```